project/plugins.sbt

```diff
@@ -5,3 +5,7 @@ addSbtPlugin("com.github.gseitz" % "sbt-release" % "1.0.3")
 addSbtPlugin("org.scoverage" % "sbt-scoverage" % "1.3.5")
 
 addSbtPlugin("org.scoverage" % "sbt-coveralls" % "1.1.0")
+
+addSbtPlugin("com.timushev.sbt" % "sbt-updates" % "0.3.0")
+
+addSbtPlugin("io.get-coursier" % "sbt-coursier" % "1.0.0-M15-1")
```

Comment:
Interesting. I had looked at Coursier in its infancy and was leery of jumping in w/ two feet at the time.